modules/installation-aws-user-infra-requirements.adoc

```diff
@@ -37,9 +39,9 @@ control plane initializes and you can access the cluster API by using the `oc`
 command line interface.
 ////
 
-You can use the following instance types for the cluster machines:
+You can use the following instance types for the cluster machines with the provided CloudFormation templates.
 
-.Valid instance types for machines
+.Instance types for machines
```

Comment:
The way that there is an important block in between this heading and the table makes it render that this title is for the note, not the table. The heading needs to be with the table - suggest moving the important admonition above the heading.